src/include/wiredtiger.in

```diff
@@ -4685,15 +4685,20 @@ extern int wiredtiger_extension_terminate(WT_CONNECTION *connection);
 #define	WT_STAT_DSRC_BLOCK_MINOR			2021
 /*! btree: btree checkpoint generation */
 #define	WT_STAT_DSRC_BTREE_CHECKPOINT_GENERATION	2022
-/*! btree: column-store fixed-size leaf pages */
+/*! btree: column-store fixed-size leaf pages, only reported if
+ * statistics=all is set */
```

Comment:
I think these instances where the message spans multiple lines should be in the format of 
/*!
 * <MESSAGE L1>
 * < MESSAGE L2>
 */